spec/fixtures/zendesk.rb

```diff
@@ -90,7 +90,9 @@ def group
 
     def organization
       VCR.use_cassette('valid_organization') do
-        @organization ||= current_user.organization
+        current_user.organization
+        # ... or, if you can't work it out locally:
+        # @organization ||= current_user.organization || client.organizations.fetch!.last
```

Comment:
There's quite some work to do when it comes to improving DX, but I'm running out of time and for now this is the best I can do.